src/os_posix/os_fs.c

```diff
@@ -748,7 +748,10 @@ __posix_open_file(WT_FILE_SYSTEM *file_system, WT_SESSION *wt_session, const cha
         f |= O_CLOEXEC;
 #endif
         WT_SYSCALL_RETRY(((pfh->fd = open(name, f, 0444)) == -1 ? -1 : 0), ret);
-        if (ret != 0)
+        /* Return error if the file not found during rollback to stable. */
+        if (ret != 0 && F_ISSET(session, WT_SESSION_ROLLBACK_TO_STABLE))
+            WT_ERR(ENOENT);
```

Comment:
What if ret isn't `ENOENT`?

Reply:
Maybe use `__wt_errno` here to figure out whether it really is `ENOENT`.

Reply:
`WT_ERR(__wt_errno())`?

Reply:
Fixed